Release docs 1.0-alpha (#295)
InterWasm/docs: docs/02-getting-started/03-setting-env.md

# Setting Up Environment

You need an environment to run contracts. You can either run your node locally or connect to an existing network. For
easy testing, the pebblenet testnet is live. You can use this to deploy and run your contracts. If you want to setup and

> run against a local blockchain, [follow these instructions](#run-local-node-optional).

Reviewer: ```suggestion
run against a local blockchain, [follow these instructions](https://docs.junochain.com/smart-contracts/installation#running-locally).
```

# Setting Up Environment

You need an environment to run contracts. You can either run your node locally or connect to an existing network. For
easy testing, the pebblenet testnet is live. You can use this to deploy and run your contracts. If you want to setup and
run against a local blockchain, [follow these instructions](#run-local-node-optional).

To verify the testnet is currently running, make sure the following URLs are all working for you:

> TODO: add deus labs

Author: Waiting for faucet setup

Author: ```suggestion
```

> - [https://rpc.uni.deuslabs.fi/status](https://rpc.uni.deuslabs.fi/status)

Reviewer: ```suggestion
- [https://rpc.uni.juno.deuslabs.fi/status](https://rpc.uni.juno.deuslabs.fi/status)
```

- [https://faucet.uni.deuslabs.fi/status](https://faucet.uni.deuslabs.fi/status)

> - [http://api.uni.deuslabs.com/node_info](http://api.uni.deuslabs.com/node_info)

Reviewer: ```suggestion
- [https://lcd.uni.juno.deuslabs.fi/node_info](https://lcd.uni.juno.deuslabs.fi/node_info)
```

> We have set up two native tokens - `STAR` (`ustar`) for becoming a validator and `SPONGE` (`ujunox`) for paying fees.

Reviewer: ```suggestion
We have set up one native token - `JUNO` (`ujunox`) for becoming a validator and for paying fees.
```

Available frontends:

// TODO: update link

> - Block Explorer: [https://block-explorer.pebblenet.cosmwasm.com](https://block-explorer.pebblenet.cosmwasm.com)

Reviewer: ```suggestion
- Block Explorer: [https://uni.junoscan.com](https://uni.junoscan.com)
```

You can use these to explore txs, addresses, validators and contracts. Feel free to deploy one pointing to our rpc/lcd
servers and we will list it.

> When interacting with this network, you can either use `wasmd` which is a Go client, or the Node REPL. The Node REPL is

Reviewer: ```suggestion
When interacting with this network, you can either use `junod` which is a Go client, or the Node REPL. The Node REPL is
```
